cmd/root.go

```diff
@@ -38,8 +38,10 @@ func NewRootCmd() *cobra.Command {
 	p := rootCmd.PersistentFlags()
 	p.BoolVar(&debug, "debug", false, "enable verbose debug logs")
 
+	genCmd := newGenerateCmd()
```

Comment:
Done see latest commit in PR: https://github.com/Azure/acs-engine/pull/957/commits/c4a7bb6f5511ac101745d868e9da9571bfd15183